extensions/wikia/EditPageLayout/js/plugins/TemplateClassificationEditorPlugin.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Editor plugin to force template classification type choose on publish if doesn't exist and break article submit
+ *
+ * @author Kamil Koterba <kamil@wikia-inc.com>
+ */
+(function (window, $) {
+		'use strict';
+
+		var WE = window.WikiaEditor = window.WikiaEditor || (new window.Observable()),
```

Comment:
I think this is the Observable used here https://github.com/Wikia/app/blob/dev/resources/wikia/jquery.wikia.js#L375-375